locations/spiders/teta_cz.py

```diff
@@ -0,0 +1,57 @@
+import json
+from typing import Iterable
+
+import scrapy
+from scrapy import FormRequest, Request, Spider
+
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class TetaCZSpider(Spider):
+    name = "teta_cz"
+    item_attributes = {"brand": "Teta", "brand_wikidata": "Q20860823"}
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def start_requests(self) -> Iterable[Request]:
+        yield FormRequest(
+            url="https://www.tetadrogerie.cz/CMSPages/Sprinx/MapData.ashx",
+            formdata={
+                "search": json.dumps(
+                    {"Extra": [], "Services": [], "CosmeticsBrands": [], "Location": {"Lat": 0, "Lng": 0}}
+                )
+            },
+            callback=self.parse_locations,
+        )
+
+    def parse_locations(self, response, **kwargs):
+        for location in response.json():
+            item = DictParser.parse(location)
+            if item["name"] == "Teta drogerie a lékárny ČR s.r.o.":
+                item["name"] = "Teta"
```

Comment:
```suggestion
            item["operator"] = item.pop("name"):
```

I only did `name=Teta drogerie a lékárny ČR s.r.o.`, if the following are better as operator, we should just move them there.

```
name=A - market s.r.o.
name=BATUL s.r.o.
name=DROGERIE A+L s.r.o.
name=DROGERIE JEVÍČKO s.r.o.
name=DROGERIE MARKET Group s.r.o.
name=Drogerie DUHA s.r.o.
name=EMESA s.r.o.
name=FESLOVÁ s.r.o.
name=Hladík Petr
name=Ing. Ngan Bui Van
name=Jiří Dohnal s.r.o.
name=Jiří Watzinger
name=Kolarevič Dušan
name=Martin Pech
name=Milan Hlavatý
name=Novysedláková Štěpánka
name=OneTech, s.r.o.
name=Paroubková Marcela
name=PeMi drogerie s.r.o.
name=Petr Doubek
name=Růžena Prokopová
name=SYNERGIC,s.r.o.
name=Susy s.r.o.
name=TETAX s.r.o.
name=Umlauf Miroslav
name=Šuška Milan
name=Šťastný s.r.o.
```